src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -62,32 +62,38 @@
 				</div>
 
 				<!-- Loan tag -->
-				<loan-tag-v2 v-if="showTags && !isLoading" :loan="loan" :amount-left="amountLeft" />
+				<router-link
+					:to="customLoanDetails ? '' : `/lend/${loanId}`"
+					v-kv-track-event="['Lending', 'click-Read more', 'Photo', loanId]"
```

Comment:
Instead of `Photo`, this should be `Tag`

Reply:
Thanks for the catch!